ui/src/app/icons/icon.tsx

```diff
@@ -1,74 +0,0 @@
-
-import colors, {colorWithWhiteness} from 'app/styles/colors';
-
-import * as React from 'react';
-
-
-const iconData = {
```

Comment:
were these used anywhere?

Reply:
These were used on the admin page only. The new font awesome icons are a little thicker in their lines. Given that it is not a public facing page I went ahead with the replacement.

Reply:
cool, thanks